api/serializers.py

```diff
@@ -0,0 +1,43 @@
+from api.models import *
+from rest_framework import serializers
+
+class ActivityExerciseSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Exercise
+        fields = ['id', 'name', 'description', 'img_url']
```

Comment:
I've added extra fields to the exercise model on my local so will also add these here (equipment and instructions)